provisioning/transport/mqtt/src/ProvisioningTransportHandlerMqtt.cs

```diff
@@ -306,6 +310,18 @@ private static DeviceRegistrationResult ConvertToProvisioningRegistrationResult(
                 websocket.Options.ClientCertificates.Add(clientCertificate);
             }
 
+            // Support for RemoteCertificateValidationCallback for ClientWebSocket is introduced in .NET Standard 2.1
+#if NETSTANDARD2_1
+            if (RemoteCertificateValidationCallback != null)
```

Comment:
Just wondering, this is supported for netstandard only then?

Reply:
Yes, the support for REmoteCertificateValidationCallback was added for websockets in .NET Standard 2.1: https://docs.microsoft.com/en-us/dotnet/api/system.net.websockets.clientwebsocketoptions.remotecertificatevalidationcallback?view=netcore-3.1